components/perf-tools/likwid/SPECS/likwid.spec

```diff
@@ -107,13 +115,24 @@ It consists of:
           COMPILER="GCC" \
           FC="gfortran" \
           FCFLAGS="-J ./ -fsyntax-only" \
+%endif
+%endif
+%ifarch aarch64
+          COMPILER="GCCARMv8" \
+          FC="gfortran" \
+          FCFLAGS="-J ./ -fsyntax-only" \
+          BUILDDAEMON="false" \
+          BUILDFREQ="false" \
+          ACCESSMODE="perf_event"
 %endif
           INSTALL_CHOWN="" \
           OPTFLAGS="%{optflags}" \
           Q="" install
 
+%ifarch x86_64
```

Comment:
At https://github.com/openhpc/ohpc/pull/1718 I've enabled BUILDDAEMON and BUILDFREQ also for the `configure` step and these binaries are also available for Linux ARM64!